### ISSUE TITLE
Problematic URLs

### ISSUE THREAD

**Issue by khromov** [ideag/content-cards#65]
A list anyone can contribute to.

**URL** http://shaminnong.blog.sohu.com/309687253.html
**Problem** Broken encoding in title field. (Probably description too if there was one.)

![test-case](https://cloud.githubusercontent.com/assets/1207507/9892663/456ea674-5c11-11e5-98f6-28cb26ab4fc2.png)

**Comment by ideag** [comment]
Yeah, thought that one will come up sooner or later. The page uses GBK (Chineese Simplified) encoding, while I assume UTF-8 for everything here: https://github.com/ideag/content-cards/blob/master/content-cards.php#L562

Too bad UTF-8 is not universally used, huh? I should probably try to detect original encoding with `mb_detect_encoding()` there.

**Comment by ideag** [comment]
Test suite for exotic encodings, etc.

```
[contentcards url="http://habrahabr.ru/post/266933/"]

[contentcards url="http://smrdv.tumblr.com/"]

[contentcards url="http://www.lookatme.ru/mag/all-mag/media-digest/113203-samoe-interesnoe-v-russkih-blogah-2010-12-28"]

[contentcards url="http://www.aftonbladet.se/nyheter/dennyaarabvarlden/syrien/article21422239.ab"]

[contentcards url="http://jp.techcrunch.com/2015/09/15/20150915opera-for-desktop-gets-password-sync-advertises-operas-surfeasy-vpn-service-as-a-privacy-option/"]

Bork

[contentcards url="http://shaminnong.blog.sohu.com/309687253.html"]

[contentcards url="http://blog.sina.com.cn/s/blog_48874cec0102vyhz.html"]

[contentcards url="http://www.aftonbladet.se/nyheter/article21396018.ab?asdasd"]

[contentcards url="http://stasoline.shrtl.com/?asdasda"]
```

**Comment by vorstera** [comment]
Hi there - please can you advise on the following 2 URL's I'm experiencing errors on:-

https://www.nets.eu/Media-and-press/news/Pages/Is-the-point-of-sale-dead.aspx

http://www.techfestconf.com/customer-experience/aus/blog/customer-engagement/can-you-tell-me-a-story/

both of them refuse to process / render - is there a limitation on what type of URL's can be used?

I've had success with all the other URL's I'm using (so far) so am just looking to find out if it's something I can resolve or if it's the nature of the target content that's preventing the creation of the cards

Many thanks

**Comment by khromov** [comment]
Got a report regarding problematic urls via e-mail. It does not seem related to https://github.com/ideag/content-cards/issues/94 as Vox runs on Fastly and El País runs on Akamai.

URL:s:
https://elpais.com/elpais/2018/05/09/album/1525884321_190028.html
https://www.vox.com/identities/2018/3/6/16929836/trans-refugees-greece-middle-east
